### DISCUSSION TITLE
Schedule...

### DISCUSSION THREAD

**Discussion by slip83** [jmcollin78/versatile-thermostat-ui-card#13] [Ideas]
Maybe I just can't find it, but the schedule seems to be missing from the settings... 
For example, at night I don't want the room stat to be triggered by motion. It would be handy to limit this to a time period... Like in traditional systems. 
Also, I have dog wandering about over night, triggering motion sensors :-)

**Comment by HACS-bank** [comment]
Did you look at the Scheduler which is mentioned in the instructions as a separate HACS component? It allows you to do what you suggest, but not within VTherm.

**Reply by slip83** [reply]
I currently use scheduler and automations for presence detection but it would be nice to have all in one place... As it stands I only use vterm for my infrared. The "over climate" function kinda defeats the purpose as I lose few capabilities of tado... All in all a good product, just sharing an idea to make it better
